### PR TITLE
new js raid 1 subject: crossword

### PR DESCRIPTION
**Pull request by nprimo**

### CODE REVIEW

File: subjects/crossword/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+The function takes two arguments:
+1. an empty puzzle, passed as a string and
+2. a list of words to fill in the puzzle (no double words allowed)
+
+The function must print on the console a string representing the puzzle filled with the input words.
+
+The empty puzzle will be a string with the following rules:
+- each character will be either a number, a `.` or a `\n`;
+- a number represent the number of words starting from the specific position and a `.` represent a space that does not need to be filled.
```
Comment (eslopfer) [inline subjects/crossword/README.md:15]: Missing 's' after represent.

File: subjects/crossword/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## crossword
+
+### Instructions
+
+Create the function `crosswordSolver` that is able to solve an empty crossword puzzle.
```
Comment (eslopfer) [inline subjects/crossword/README.md:5]: What would you think about specifying that the file needs to have that name as well?

File: subjects/crossword/audit/README.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+```
+
+```
+casa
+i..l
+anta
+o..n
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:24]: Display without 's'

File: subjects/crossword/audit/README.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+.m.....seaside.
+.m.b....a.a....
+.icecream.n....
+.n.a......d....
+.g.c.....tan...
+...h......l....
+..........s....
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:74]: Same as above

File: subjects/crossword/audit/README.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:76]: Same as above

File: subjects/crossword/audit/README.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+..o.t..t...
+pork..pasta
+..n.s..b...
+....t..l...
+..cheese...
+....a..s...
+....k......
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:120]: Same as above

File: subjects/crossword/audit/README.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+```js
+const puzzle = '2001\n0..0\n1000\n0..0'
+const words = ['casa', 'casa', 'ciao', 'anta']
+```
+
+```
+Error
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:133]: Same as above

File: subjects/crossword/audit/README.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+```js
+const puzzle = '0001\n0..0\n3000\n0..0'
+const words = ['casa', 'alan', 'ciao', 'anta']
+```
+
+```
+Error
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:146]: Same as above

File: subjects/crossword/audit/README.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+```js
+const puzzle = '2001\n0..0\n1000\n0..0'
+const words = ['casa', 'casa', 'ciao', 'anta']
+```
+
+```
+Error
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:159]: Same as above

File: subjects/crossword/audit/README.md [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+```js
+const puzzle = ''
+const words = ['casa', 'alan', 'ciao', 'anta']
+```
+
+```
+Error
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:172]: Same as above

File: subjects/crossword/audit/README.md [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+```js
+const puzzle = 123
+const words = ['casa', 'alan', 'ciao', 'anta']
+```
+
+```
+Error
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:185]: Same as above

File: subjects/crossword/audit/README.md [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+```js
+const puzzle = ''
+const words = 123
+```
+
+```
+Error
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:198]: Same as above

File: subjects/crossword/audit/README.md [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+```js
+const puzzle = '2001\n0..0\n1000\n0..0'
+const words = ['aaab', 'aaac', 'aaad', 'aaae']
+```
+
+```
+Error
+```
+
+###### Does the function displays the value above?
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:211]: Same as above

File: subjects/crossword/audit/README.md [hunk 14]
```diff
@@ -0,0 +1,215 @@
...
+
+#### Bonus
+
+###### +Is the project using ...
```
Comment (eslopfer) [inline subjects/crossword/audit/README.md:215]: I guess this is because it's not finished yet? :smiley:

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: subjects/crossword/audit/README.md (hunk 14 above)
Comment (nprimo) [reply subjects/crossword/audit/README.md:215]: Correct! I will add some option but if you have any suggestions please, let me know :)

**Comment by nprimo** [comment]
I have fixed the issues you pointed out and I have added a small suggestion for the bonus section in the audit

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**